<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>volunteerprofile</class>
 <widget class="QWidget" name="volunteerprofile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>volunteerprofile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(20, 181, 255), stop:1 rgb(255, 255, 255, 255))</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>-40</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
  <widget class="QPushButton" name="pushlogout">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="collection">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Collection</string>
   </property>
  </widget>
  <widget class="QLabel" name="sumtotal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>107</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
   </property>
   <property name="text">
    <string>Total hours :</string>
   </property>
  </widget>
  <widget class="QLabel" name="filler1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="sumpoint">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>99</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
   </property>
   <property name="text">
    <string>Your Point :</string>
   </property>
  </widget>
  <widget class="QLabel" name="filler2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>530</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>310</y>
     <width>691</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcomemsg">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>187</width>
     <height>94</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground),
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Welcome,&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
   </property>
   <property name="text">
    <string>Pts</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
   </property>
   <property name="text">
    <string>hrs</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>91</y>
     <width>108</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>91</y>
     <width>42</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground)</string>
   </property>
   <property name="text">
    <string>progress</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcomemsg_3">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>187</width>
     <height>94</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground),
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;&amp;lt;name&amp;gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="welcomemsg_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>591</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:setAttribute(Qt::WA_TranslucentBackground),
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;lt;msg&amp;gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushprofile">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>64</width>
     <height>46</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QLCDNumber" name="ptsIcd">
    <property name="styleSheet">
     <string notr="true">background: black</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="hrsIcd">
    <property name="styleSheet">
     <string notr="true">background: black</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
